integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/StandardBusinessDocumentFactory.java

```diff
@@ -89,13 +96,46 @@ public EduDocument create(EDUCore shipment, String conversationId, Avsender avse
         return new CreateSBD().createSBD(avsender.getOrgNummer(), mottaker.getOrgNummer(), payload, conversationId, DOCUMENT_TYPE_MELDING, shipment.getJournalpostId());
     }
 
+    public EduDocument create(ConversationResource shipmentMeta, MessageContext context) throws MessageException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: no.difi.meldingsutveksling.noarkexchange.MessageException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1160)